docs/_includes/ds6/select.html

```diff
@@ -0,0 +1,70 @@
+<div id="select">
+    <h2><span class="secondary-text">@ebay/skin/</span>select</h2>
+
+    <h3>Default select</h3>
+
+    <div class="demo">
+        <div class="demo__inner" style="max-width: 200px;">
+            <span class="select">
+                <select name="nativeOptions">
+                    <option value="item1">Option 1</option>
+                    <option value="item2">Option 2</option>
+                    <option value="item3">Option 3</option>
+                </select>
+            </span>
+        </div>
+        {% highlight html %}
+<span class="select">
+    <select name="nativeOptions">
```

Comment:
Being picky, but let's change the name to just `options`.

Reply:
⛏